moduleframework/common.py

```diff
@@ -347,10 +353,9 @@ def runHost(self, command="ls /", **kwargs):
             formattedcommand = command.format(**trans_dict)
         except KeyError:
             raise ModuleFrameworkException(
-                "Command is formatted by using trans_dict. If you want to use "
-                "brackets { } in your code, please use {{ }}. Possible values "
-                "in trans_dict are: %s. \nBAD COMMAND: %s"
-                % (trans_dict, command))
+                "Command is formatted by using trans_dict, if you want to use brackets { } in your code please use {{ "
+                "or }}, possible values in trans_dict are:",
+                trans_dict, "BAD COMMAND:", command)
         return process.run("%s" % formattedcommand, **kwargs)
```

Comment:
This is a wrong string update. Check with devel.

Reply:
This was part of another PR. I don't want to fix it in this PR.